src/graph/parser.ts

```diff
@@ -135,6 +137,25 @@ export class Graph {
         }
     }
 
+    private static parseSize(cssString: string):{value: string, isDynamic: boolean} {
+        cssString = cssString.toString()
+        let index = cssString.search(/[A-Za-z%]/)
```

Comment:
Not sure what this is for, is this to match the trailing unit? If so you should probably be only matching the end of the string, and we should be validating against a list of allowed css values to prevent an invalid input from producing a confusing result.